contrib/hashicorp/vault/vault.go

```diff
@@ -0,0 +1,98 @@
+// The vault package contains functions to construct an *http.Client that will
+// integrate with the github.com/hashicorp/vault/api and collect traces to send
+// to DataDog.
+//
+// The easiest way to use this package is to create an *http.Client with
+// NewHttpClient, and put it in the vault api config that's passed to the
+// NewClient function.
+//    import "github.com/hashicorp/vault/api"
+//    import trace "gopkg.in/DataDog/dd-trace-go.v1/contrib/hashicorp/vault"
+//
+//    func ... {
+//        api.NewClient(&api.Config{HttpClient: trace.NewHttpClient()})
+//    }
+//
+// If you are already using your own *http.Client with vault, you can use the
+// WrapHttpTransport function to wrap the client with the tracer code. Your
+// *http.Client will continue to work as before, but will also capture traces.
+//
+//    func ... {
```

Comment:
My recommendation is to instead just create examples for both of these use cases instead of writing invalid code containing ellipsis (`...`). They will be clear enough.